.github/workflows/build-artifacts.yml

```diff
@@ -11,10 +11,7 @@ on:
 
 jobs:
   build:
-    strategy:
-      matrix:
-        os: [ubuntu-latest, macos-latest]
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-latest
```

Comment:
Since we're not building binary files anymore, we don't need to run this on multiple systems.